sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -19,7 +19,7 @@
 
 from sklearn.utils._testing import (assert_almost_equal, assert_array_equal,
                                     assert_array_almost_equal,
-                                    assert_allclose,
+                                    assert_allclose, assert_no_warnings,
```

Comment:
Maybe add a line break here to respect the structure of the imports.

Reply:
Thanks, I added a line break.